backend/azureblob/azureblob.go

```diff
@@ -10,17 +10,22 @@ import (
 	"crypto/md5"
 	"encoding/base64"
 	"encoding/hex"
+	"encoding/json"
 	"fmt"
 	"io"
+	"io/ioutil"
 	"net/http"
 	"net/url"
 	"path"
 	"strings"
 	"sync"
 	"time"
 
+	"github.com/rclone/rclone/lib/env"
+
```

Comment:
Can you remove this blank line and put the added import in sorted order (likely your editor will do this)